contracts/seed/SeedFactory.sol

```diff
@@ -44,35 +40,38 @@ contract SeedFactory is CloneFactory {
     }
 
     modifier protected() {
-        require(initialized,                    "SeedFactory: contract not initialized");
-        require(msg.sender == address(avatar),  "SeedFactory: protected operation");
+        require(initialized, "SeedFactory: contract not initialized");
+        require(
+            msg.sender == address(avatar),
```

Comment:
I think `protected` would be better split into two separate functions, "adminOnly" and "isInitialized"

Reply:
I've made these changes, but it is not efficient to use more modifiers if it can be done in one modifier.

Reply:
Coding design pattern:   single-responsibility principle